app/views/articles/_actions.html.erb

```diff
@@ -20,8 +20,19 @@
     </button>
     <div class="dropdown-content">
       <div>
+        <script async>
+          function copyLink() {
+            var copyButton = document.getElementById('copy-link')
+            copyButton.select()
```

Comment:
Ah my bad, didn't have access to an older iPhone. Will download a simulator and test. What's the lowest iOS version/phone version that we cater to?

Reply:
We don't officially have an iOS version that we support, but I imagine that fixing it would make it work on most lower versions. It'll be great if this works with iOS 11.x.x, since it's the latest version.